### ISSUE TITLE
the model based in tf-unet doesn't work

### ISSUE THREAD

**Issue by 541435721** [shiba24/3d-unet#4]
comparing your model.py with tf_unet/unet.py, i find that you just changed the names of conv2d and deconv2d. And the sizes of  convolution kernels  are never changed. Accrording to the official apis, the filter size of conv3d must be 5-D.

**Comment by shiba24** [comment]
@541435721 yes, tensorflow-base implementation is work in progress. Chainer version can work now.
